src/libsystemd-network/sd-ipv4acd.c

```diff
@@ -71,14 +71,19 @@ struct sd_ipv4acd {
         IPv4ACDState state;
         int ifindex;
         int fd;
-        int iteration;
-        int conflict;
+
+        unsigned n_iteration;
+        unsigned n_conflict;
```

Comment:
well, iteration, does, but conflict does not...

I usually used the "n_" prefix whenever you count something, i.e. you regularly see a line like `n_foo++;` in place